application/models/Execution.php

```diff
@@ -23,6 +23,16 @@ function find($id)
         return $this->db->query("SELECT * FROM $this->table WHERE id = ?;", [$id])->row();
     }
 
+    public function insert($data)
+    {
+        $this->db->insert($this->table, $data);
+        return $this->db->insert_id();
```

Comment:
Please leave a blank line before the return (PSR-2 https://www.php-fig.org/psr/psr-2/)

Reply:
Done (damn PSR)